StRoot/StFttClusterMaker/StFttClusterMaker.cxx

```diff
@@ -38,7 +38,7 @@ StFttClusterMaker::StFttClusterMaker( const char* name )
   mDebug( false ),       /// print out of all full messages for debugging
   mFttDb( nullptr )
 {
-    LOG_DEBUG << "StFttClusterMaker::ctor"  << endm;
+    LOG_INFO << "StFttClusterMaker::ctor"  << endm;
```

Comment:
Is LOG_INFO really appropriate here?

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec